### PR TITLE
Installing eslint-prettier-config and adding it to .eslintrc

### PR DESCRIPTION
**Pull request by minggas**
Install `eslint-config-prettier` and add `"prettier"` to `"extends"` on `.eslintrc` to solve #130

### CODE REVIEW

**Comment by minggas** [comment]
What was the problem?

**Comment by Janther** [comment]
Relax @minggas nothing is wrong with this (but I know little about eslint's configurations).
The coverage didn't pass because of `src/clean.js`
We are discussing it in #135

**Comment by mattiaerre** [comment]
you should be able to either rebase or create another pr from master @minggas now that #151 has been merged

File: .eslintrc [hunk 1]
```diff
@@ -2,7 +2,10 @@
   "env": {
     "jest": true
   },
-  "extends": "airbnb",
+  "extends": [
+    "airbnb",
```
Comment (mattiaerre) [inline .eslintrc]: should we be also getting rid of `airbnb`?

Comment (fvictorio) [reply .eslintrc]: I don't think so, it has some useful rules.

**Comment by fvictorio** [comment]
This looks fine. My only comment is that I think now we can remove the rules that we have explicitly disabled in the configuration.

**Comment by mattiaerre** [comment]
I am going to merge this a create a followup PR w/ some more cleanup

**Pull request merged**